app/src/main/res/layout/user_profile_screen.xml

```diff
@@ -0,0 +1,202 @@
+        android:layout_width="match_parent"
+        android:layout_height="60dp"
+        app:cardElevation="16dp"
+        tools:targetApi="lollipop"
+        >
+
+        <androidx.appcompat.widget.AppCompatImageView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_vertical"
+            android:src="@drawable/ic_arrow_back_blue_24dp"
+            />
+
+        <androidx.appcompat.widget.AppCompatTextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_vertical"
+            android:layout_marginLeft="20sp"
+            android:layout_marginStart="40dp"
+            android:text="@string/profile_string"
+            android:textColor="@color/colorBlue"
+            android:textSize="24sp"
+            />
+    </androidx.cardview.widget.CardView>
+
+    <androidx.cardview.widget.CardView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:layout_marginTop="70dp"
+        app:cardElevation="0dp"
+        >
+
+        <androidx.appcompat.widget.LinearLayoutCompat
+            android:layout_width="150dp"
+            android:layout_height="137dp"
+            android:gravity="center"
+            >
+
+            <androidx.appcompat.widget.AppCompatImageView
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:contentDescription="@string/user_profile_image_string"
+                android:src="@drawable/ic_user_smal"
+                />
+
+        </androidx.appcompat.widget.LinearLayoutCompat>
+
+        <androidx.cardview.widget.CardView
+            android:layout_width="45dp"
+            android:layout_height="45dp"
+            android:layout_gravity="end"
+            android:layout_marginEnd="2dp"
+            android:layout_marginRight="4dp"
+            android:layout_marginTop="16dp"
+            app:cardCornerRadius="40dp"
+            >
+
+            <androidx.appcompat.widget.AppCompatImageView
+                android:layout_width="35dp"
+                android:layout_height="35dp"
+                android:layout_gravity="center"
+                android:contentDescription="@string/camera_icon_string"
+                android:padding="3dp"
+                android:src="@drawable/ic_camera"
+                />
+        </androidx.cardview.widget.CardView>
+    </androidx.cardview.widget.CardView>
+
+    <androidx.cardview.widget.CardView
+        android:layout_width="300dp"
+        android:layout_height="84dp"
+        android:layout_gravity="center"
+        android:layout_marginTop="100dp"
+        android:padding="20dp"
+        app:cardElevation="0dp"
+        >
+
+        <androidx.appcompat.widget.LinearLayoutCompat
+            android:layout_width="260dp"
```

Comment:
Don't use hard-coded values.

Reply:
@iadeelzafar I have to either use hardcoded values for layout width of the Linear layout or the Edit text, for the UI to look exactly like the Design. What should I do now?